src/main/java/com/example/ludogoriesoft/lukeriaerpapi/controllers/ProductController.java

```diff
@@ -33,12 +34,12 @@ public ResponseEntity<ProductDTO> createPlate(@Valid @RequestBody ProductDTO pro
     }
 
     @PutMapping("/{id}")
-    public ResponseEntity<ProductDTO> updateProduct(@PathVariable("id") Long id, @Valid @RequestBody ProductDTO productDTO) {
+    public ResponseEntity<ProductDTO> updateProduct(@PathVariable("id") Long id, @Valid @RequestBody ProductDTO productDTO) throws ChangeSetPersister.NotFoundException {
```

Comment:
Since we are working with the `ProductDTO` here I believe that the validation annotations like `@Min` should be added to the DTO and not to the model

Reply:
i added validate in DTOs